www/app/main/main.controller.js

```diff
@@ -0,0 +1,10 @@
+(function() {
+  angular
+    .module('app')
+    .controller('mainController', mainController);
```

Comment:
Capitalize MainController and the function MainController per the JP Style Guide